src/components/Accordion/Accordion.spec.tsx

```diff
@@ -266,15 +265,31 @@ describe('Components / Accordion', () => {
       expect(content()[1]).not.toBeVisible(); // content should not be visible
     });
   });
+
+  describe('Collapse/expand panel', () => {
```

Comment:
Please just collapse this into a single test like

*should collapse panel if `isOpen` and expand if not after a user toggle event*

Instead of introducing a `beforeEach` here